lxd/container_lxc.go

```diff
@@ -1115,6 +1115,10 @@ func (c *containerLXC) Start(stateful bool) error {
 		wgStopping.Wait()
 	}
 
+	if err := setupSharedMounts(); err != nil {
+		return fmt.Errorf("Daemon failed to setup shared mounts base: %s", err)
```

Comment:
That's where we should include a hint that it could be because the container isn't allowed to run nested containers.